src/main/java/net/sf/jabref/gui/fieldeditors/TextAreaForVisibleField.java

```diff
@@ -0,0 +1,25 @@
+package net.sf.jabref.gui.fieldeditors;
+
+import java.util.Objects;
+
+/**
+ * Indicates that the field of this TextArea can be hidden
```

Comment:
Please address the comment. Whether write why you would leave this as it is or change it, but please dont let it unanswered.